azure/iac/sql_servers.rego

```diff
@@ -91,12 +95,20 @@ sql_server_login = false {
 }
 
 sql_server_login = false {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.sql/servers"
+    sql_resources := resource.resources[_]
+    lower(sql_resources.type) == "administrators"
```

Comment:
it is necessary because:
```no_azure_issue["sql_server_login"] {
    resource := input.resources[_]
    lower(resource.type) == "microsoft.sql/servers"
    sql_resources := resource.resources[_]
    lower(sql_resources.type) == "administrators"
    lower(sql_resources.properties.login) != "admin"
    lower(sql_resources.properties.login) != "administrator"
}
```
 ```no_azure_issue``` can be false by each one of this condition:
for example: if ```lower(resource.type) == "microsoft.sql/servers"``` was false or ```lower(sql_resources.properties.login) != "admin"``` was false

in the first case, we need to skip the test and in the second case, it should be false
so 
```
sql_server_login = false {
    resource := input.resources[_]
    lower(resource.type) == "microsoft.sql/servers"
    sql_resources := resource.resources[_]
    lower(sql_resources.type) == "administrators"
    not no_azure_issue["sql_server_login"]
}
```
we want ```sql_server_login ``` be false if the type of resource is correct